app/main.py

```diff
@@ -85,129 +87,22 @@
     }
 }
 
-
 def battle(knightsConfig):
     # BATTLE PREPARATIONS:
+    lancelot = Knight(knightsConfig["lancelot"])
+    arthur = Knight(knightsConfig["arthur"])
+    mordred = Knight(knightsConfig["mordred"])
+    red_knight = Knight(knightsConfig["red_knight"])
```

Comment:
Ensure that all expected keys exist in the `knightsConfig` dictionary to avoid potential KeyErrors when creating Knight instances. Consider adding validation or default values in the `Knight` class to handle missing keys gracefully.